### PR TITLE
Disabled states for buttons

### LINKED ISSUE #36: `:disabled` state for buttons
Currently the buttons don't appear disabled when you add a disabled attribute to them. They appear and interact with the mouse as though they are clickable.

### PR DESCRIPTION
**Pull request by 1nusah**
Resolves #36

### CODE REVIEW

**Comment by blackmann** [comment]
Thanks for this.

Couple of recommendations:

1. Let’s place the disabled button under a section. You see how the inputs are sectioned; something like that. So a header like “Disabled state” below the previous three buttons
2. Also let’s maintain the import order. I’m surprised eslint isn’t catching that.

**Comment by 1nusah** [comment]
>
Sure. Does this work?
image

File: app/src/css.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import fontIndex from './gfonts'
 import { getStyle } from './config'
+import fontIndex from './gfonts'
```
Comment (blackmann) [inline app/src/css.ts:2]: Should come before `import { getStyle } …`

Comment (1nusah) [reply app/src/css.ts]: Fixed

**Comment by blackmann** [comment]
Thanks for this mann. Want a mention on Twitter? What’s your handle?

**Pull request merged**